### ISSUE TITLE
learning_rate not changing on monitoring.

### ISSUE THREAD

**Issue by areche** [tensorpack/tensorpack#1262]
Hi! 

I'm using the FasterRCNN example with this config changes :

```
MODE_FPN=True 
FPN.CASCADE=True 
BACKBONE.RESNET_NUM_BLOCKS=[3,4,23,3] 
FPN.NORM=GN 
BACKBONE.NORM=GN 
FPN.FRCNN_HEAD_FUNC=fastrcnn_4conv1fc_gn_head
FPN.MRCNN_HEAD_FUNC=maskrcnn_up4conv_gn_head 
PREPROC.TRAIN_SHORT_EDGE_SIZE=[640,800] 
TRAIN.LR_SCHEDULE=[1250000,1500000,1750000] 
BACKBONE.FREEZE_AT=0 
TRAIN.STEPS_PER_EPOCH=2000
```

and while the output of the callbacks says that the learning rate will be changed:

```
[0711 12:39:32 @base.py:275] Start Epoch 1250 ...
100%|########################################|2000/2000[17:20<00:00, 1.91it/s]
[0711 12:56:53 @base.py:285] Epoch 1250 (global_step 1538000) finished, time:17 minutes 20 seconds.
[0711 12:56:53 @graph.py:73] Running Op sync_variables/sync_variables_from_main_tower ...
[0711 12:56:54 @saver.py:79] Model saved to train_log/maskrcnn/model-1538000.
[0711 12:56:54 @param.py:158] [HyperParamSetter] At global_step=1538000, learning_rate is set to 0.001000
[0711 12:56:58 @misc.py:109] Estimated Time Left: 6 days 1 hour 6 minutes 20 seconds
[0711 12:56:58 @eval.py:294] Running evaluation ...
```

The monitoring and Tensorboard keeps showing the previous learning rate:

`[0711 14:14:16 @monitor.py:467] learning_rate: 0.01`

Is that supposed to work this way? Is this a known problem?

Thanks,

Alex

**Comment by areche** [comment]
My mistake... I was too quick... the learning rate when down as expected in the epoch 1251.

Thanks for a great work!
